lib/kalibro_client/entities/processor/metric_collector_details.rb

```diff
@@ -18,23 +18,28 @@ module KalibroClient
   module Entities
     module Processor
       class MetricCollectorDetails < KalibroClient::Entities::Processor::Base
-        attr_accessor :name, :description, :supported_metrics, :wanted_metrics, :processing
```

Comment:
does not sound good to remove the supported_metrics from accessor. I guess that prezento relies on it...